tempest/matrix/C33Matrix.hpp

```diff
@@ -0,0 +1,42 @@
+#include "tempest/vector/C3Vector.hpp"
+
+#ifndef TEMPEST_MATRIX_C_33MATRIX_HPP
+#define TEMPEST_MATRIX_C_33MATRIX_HPP
+
+class C33Matrix {
+    public:
+    // Member variables
+    float a0 = 0.0f;
+    float a1 = 0.0f;
+    float a2 = 0.0f;
+    float b0 = 0.0f;
+    float b1 = 0.0f;
+    float b2 = 0.0f;
+    float c0 = 0.0f;
+    float c1 = 0.0f;
+    float c2 = 0.0f;
```

Comment:
From what I can tell, it appears `C33Matrix`'s default constructor should set up an identity matrix. Thus, the member initializers should be:

```cpp
float a0 = 1.0f;
// snip
float b1 = 1.0f;
// snip
float c2 = 1.0f;
```